front/app/containers/Admin/invitations/messages.ts

```diff
@@ -218,4 +218,8 @@ export default defineMessages({
     id: 'UI.FormComponents.required',
     defaultMessage: 'required',
   },
+  searchForUsers: {
```

Comment:
Ah, I see! Is it because you wanted to add this custom copy? In the search component the label just says search but I think it's clear enough. Also - I recommend using it because it includes some additional things like throttling for example.

Reply:
@IvaKop Clicking on the search field said "Edit text" three times using my screen reader, but nothing with "search" unfortunately. Or are you referring to the SearchInput component? Should I replace the currently used Input with SearchInput?

Reply:
Ah - just saw your review comment below. I'll swap in the SearchInput component :+1:

Reply:
@IvaKop If I replace the Input with the SearchInput component, should the onChange function that is called (  `handleChangeSearchTerm = (event) =>` ) use a callback instead (`const handleChangeSearchTerm = useCallback((newSearch: string) =>` )? I'm getting an error `Cannot read properties of null (reading 'target') ` with the original onChange function, as there is no event object anymore.

Reply:
Seems like the search term is a string, so you can use it directly, instead of the event.